bitrise.yml

```diff
@@ -253,22 +252,24 @@ workflows:
             #!/usr/bin/env bash
             set -e
             make test-cocoapods ci=${CI}
-    - xcodebuild:
-        title: Build CPProject for tests - Catalyst
```

Comment:
Here I remove the Mac Catalyst build for Cocoapods from `bitrise.yml`. Mac Catalyst compatibility is already tested in the previous step, when building with Swift Package Manager. IMO testing it once again for Cocoapods is redundant as if it worked for SPM, it must for for CP too (nothing had to be configured in `Package.swift` nor `podspec` when enabling it in https://github.com/DataDog/dd-sdk-ios/pull/277).

Removing this step gives us additional CI time, now used for testing _no `use_frameworks!` compatibility for CP.